### PR TITLE
Expo common issue - Unable to resolve assets

### PR DESCRIPTION
**Pull request by brunokiafuka**
The issue normally happens after the SDK update.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,3 +68,15 @@ import {KeyboardAvoidingView, Platform} from 'react-native';
...
 </KeyboardAvoidingView>;
 ```
+
+### **Unable to resolve assets**
+
+- Após a actualização do SDK do expo as _**custom fonts**_  ou _**icons importados**_ da biblioteca `@expo/vector-icons` ocorre um erro marcando que o app não consegue resolver o path destes assets. Resolução: 
+
+```
+apagar a pasta node_modules.
+apagar package-lock.json (yarn.lock if you are using yarn instead of npm).
```
Comment (fhugoduarte) [inline README.md:78]: the default language is pt-br, so please follow it

**Review by fhugoduarte — COMMENTED** [review]
Could you send more details about this error?
I don't remember it, so I'm looking for.

An print will help me a lot. :)

**Comment by brunokiafuka** [comment]
> Could you send more details about this error?
> I don't remember it, so I'm looking for.
> 
> An print will help me a lot. :)

Hi @fhugoduarte, I'll recreate the error and send screenshots. Sorry for the delay in my response.

File: README.md (hunk 1 above)
Comment (brunokiafuka) [reply README.md:78]: should I say something like `deletar o ficheiro ...`?

**Comment by brunokiafuka** [comment]
> Could you send more details about this error?
> I don't remember it, so I'm looking for.
> 
> An print will help me a lot. :)

@fhugoduarte, I just updated my project to the expo SDK to 36 and the issue that I was describing on my comment. 

Simulator Screen Shot - iPhone 11 - 2019-12-10 at 13 27 58

**Comment by fhugoduarte** [comment]
@brunokiafuka Hi! I understand, basically you need to update the version of vector-icons, but I think this problem will not be reproduced by most people, because when creating a new expo app, it will be using the most current sdk, what do u think?

**Comment by brunokiafuka** [comment]
Well, it happens mostly if someone updates the SDK and uses custom fonts. This issue only reproduces if the user does an update on their SDK. 

The main reason is that I mentioned it is that Expo updates their SDK quite often.

**Comment by fhugoduarte** [comment]
As this isn't a common problem and the solution is quite simple, I won't add it, but thank you very much! 💜

**Pull request closed without merging**